site/en/start/go.md

```diff
@@ -0,0 +1,343 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# Bazel Tutorial: Build a Go Project
+
+{% include "_buttons.html" %}
+
+This tutorial introduces you to the basics of Bazel by showing you how to build a Go (Golang) project. You'll learn how to set up your workspace, build a small program, import a library, and run its test. Along the way, you'll learn key Bazel concepts, such as targets and `BUILD` files.
+
+Estimated completion time: 30 minutes
+
+## What you'll learn
```

Comment:
Is there actually missing content for `## What you'll learn`?

Reply:
Sorry, I meant to take that heading out, now done. I initially followed some of the structure from the Java tutorial, but then it seemed better to just write the things you'll learn in the first paragraph.

Reply:
Np!